### PR TITLE
Updated "Add-ons API Ref" as private preview

### PR DESCRIPTION
**Pull request by ashishnirvana1**
**Rendered article for review:**
https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/api/addons-api-reference?branch=pr-en-us-1705

Made a few changes to indicate that this API is available in private preview hence not be visible to everyone.  

Writer/Editor pass:
vchapel

### CODE REVIEW

File: microsoft-edge/extensions-chromium/publish/api/addons-api-reference.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 ms.topic: conceptual
 ms.prod: microsoft-edge
 ms.date: 08/19/2021
 ---
-# Microsoft Edge Add-ons API Reference (under development)
+# Microsoft Edge Add-ons API Reference (in private preview)
 
 > [!NOTE]
-> This article is a Request for Comments.  The Microsoft Edge Add-ons API is not yet available for testing, and the Publish APIs page is not yet available at Partner Center.  The Microsoft Edge Add-ons API is under active development and the roadmap continues to evolve based on market changes and customer feedback.  The plans outlined here are not exhaustive and are subject to change.
+> This article is a Request for Comments.  The Microsoft Edge Add-ons API is currently in private preview, and the Publish APIs page is not yet available at Partner Center for everyone.  The Microsoft Edge Add-ons API is under active development and the roadmap continues to evolve based on market changes and customer feedback.  The plans outlined here are not exhaustive and are subject to change.
```
Comment (mikehoffms) [inline microsoft-edge/extensions-chromium/publish/api/addons-api-reference.md:13]:
```suggestion
> The Microsoft Edge Add-ons API is currently in private preview. The **Publish API** page is present at Partner Center only for participants of the private preview. The Microsoft Edge Add-ons API is under active development, and the roadmap continues to evolve based on market changes and customer feedback. The plans outlined here aren't exhaustive, and are subject to change.
```

File: microsoft-edge/extensions-chromium/publish/api/addons-api-reference.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 ms.topic: conceptual
 ms.prod: microsoft-edge
 ms.date: 08/19/2021
 ---
-# Microsoft Edge Add-ons API Reference (under development)
+# Microsoft Edge Add-ons API Reference (in private preview)
 
 > [!NOTE]
-> This article is a Request for Comments.  The Microsoft Edge Add-ons API is not yet available for testing, and the Publish APIs page is not yet available at Partner Center.  The Microsoft Edge Add-ons API is under active development and the roadmap continues to evolve based on market changes and customer feedback.  The plans outlined here are not exhaustive and are subject to change.
+> This article is a Request for Comments.  The Microsoft Edge Add-ons API is currently in private preview. The **Publish API** page is present at Partner Center only for participants of the private preview. The Microsoft Edge Add-ons API is under active development, and the roadmap continues to evolve based on market changes and customer feedback. The plans outlined here aren't exhaustive, and are subject to change.
```
Comment (mikehoffms) [inline microsoft-edge/extensions-chromium/publish/api/addons-api-reference.md:13]: PR 1704 & PR 1705 should both have this statement, or neither PR should have it:
"This article is a Request for Comments."

Comment (mikehoffms) [inline microsoft-edge/extensions-chromium/publish/api/addons-api-reference.md:13]:
```suggestion
> The Microsoft Edge Add-ons API is currently in private preview.  The **Publish API** page is present at Partner Center only for participants of the private preview.  The Microsoft Edge Add-ons API is under active development, and the roadmap continues to evolve based on market changes and customer feedback.  The plans outlined here aren't exhaustive, and are subject to change.
```

**Review by mikehoffms — APPROVED** [review]

**Review by vchapel — APPROVED** [review]
LGTM

**Pull request merged**